packages/adapter-drizzle/src/lib/mysql.ts

```diff
@@ -1,29 +1,29 @@
 import { and, eq } from "drizzle-orm"
 import {
-  int,
-  timestamp,
-  primaryKey,
-  varchar,
+  MySqlColumn,
   MySqlDatabase,
-  mysqlTable,
   MySqlTableWithColumns,
-  TableConfig,
-  QueryResultHKT,
   PreparedQueryHKTBase,
-  index,
+  QueryResultHKT,
+  int,
+  mysqlTable,
+  primaryKey,
+  text,
+  timestamp,
+  varchar,
 } from "drizzle-orm/mysql-core"
 
 import type {
   Adapter,
-  AdapterUser,
   AdapterAccount,
   AdapterSession,
+  AdapterUser,
   VerificationToken,
 } from "@auth/core/adapters"
 
 import { randomUUID } from "crypto"
```

Comment:
i think you wanna use the global crypto module instead of importing to support as many runtimes as possible:

```ts
// instead of this
import { randomUUID } from "crypto";
const id = randomUUID()

// do this
const id = crypto.randomUUID()
```

Reply:
Currently can't use Turbopack because of this.

Reply:
This is fixed in this PR